data/src/main/java/io/gnosis/data/adapters/TokenTypeAdapter.kt

```diff
@@ -13,7 +13,7 @@ class TokenTypeAdapter {
     @FromJson
     fun fromJson(tokenType: String): TokenType =
         when (tokenType) {
-            "ETHER" -> TokenType.ETHER
+            "ETHER" -> TokenType.NATIVE_CURRENCY
```

Comment:
We need a FIXME here and on the client gw side it has to be adjusted also. Something like "NATIVE" would be nice. Could you create an issue for client gw please?

Reply:
I think there is already a ticket. But I am not sure it covers this: https://github.com/gnosis/safe-client-gateway/issues/179

Reply:
I added a FIXME